episodes/formative-assessment.md

```diff
@@ -51,30 +51,31 @@ Useful further reading: [Course design: Considerations for Trainers – a Profes
+The most effective way to test learner understanding and progress is to do such assessments in class - they engage all learners and allow instructors to check learners' confidence with the content and its delivery, can help you deal with any potential misunderstandings as soon as they arise, and maximise the value of workshop for everyone. Such formative assessments also help with *metacognition* - the awareness a learner has that they are succeeding in learning something new.
 
 Any instructional tool that generates feedback and is used in a formative way to check for learners' understanding can be described as “formative assessment”. For example,
 
-- **reflection** at the end of a session to help process learning - e.g. asking learners to write down things they learned, things they want to know more about and any questions they still have
-- **concept maps and diagrams** - asking learners to reflect by drawing/labeling a concept map/diagram or writing down a list of new concepts and skills they’ve learned and (optionally) how they relate to one another or connect with previous knowledge
 - **checking in** - gauging learners’ satisfaction and understanding using agreed signals (e.g. raising different coloured post-it/sticky notes or [Zoom reactions](https://coderefinery.github.io/manuals/zoom-mechanics/#reactions) to indicate that the pace is too fast/slow, that they completed/have not completed an exercise).
-- **think, pair, share** - learners _think_ about an answer to a question, _pair_ up
-with a classmate to discuss their answer, and then _share_ out the consensus they came to
+- **group or collaborative discussions** - raising questions for discussion among the group. For example, **think, pair, share**, where learners _think_ individually about an answer to a question, _pair_ up with a classmate to discuss their answer, and then _share_ out the consensus they came to
 with the class.
-- **exercises** - setting challenges for learners to tackle and raising questions for discussion among the group. For example, Parson's Problems, which we will learn more about in the next exercise.
+- **problem-solving or diagnostic exercises** - setting challenges for learners to tackle, testing comprehension of a subject and diagnosing any forming misconceptions.
+- **reflections** at the end of a session to help process what was learned, for example:
+    - asking learners to write down things they learned, things they want to know more about and any questions they still have or using a specific worksheet or journal for learners to summarise their thoughts about what they learned, or
+    - asking learners to reflect by drawing or labelling a concept map or a diagram or writing down a list of new concepts and skills they’ve learned and (optionally) how they relate to one another or connect with previous knowledge.
```

Comment:
I'm not sure this needs 2 bullet points, which makes it longer than the others.  Could we maybe combine write, draw, or diagram into one shorter sentence?  It could cut out the specifics about worksheets/journals/concept maps etc though.
```suggestion
- **reflections** at the end of a session to help process what was learned, for example asking learners to write down, draw, or diagram (concept map) things they learned, noting things they want to know more about and any questions they still have.
```

Reply:
Agreed - 2 bullet points not needed.